src/guide/migration/render-function-api.md

```diff
@@ -128,6 +128,47 @@ In 3.x, the entire VNode props structure is flattened. Using the example from ab
 }
 ```
 
+## Registered Component
+
+### 2.x Syntax
+
+In 2.x, when a component has been registered, the render function will work well when passing the component's name as string to the first arguments:
+
+```js
+// 2.x
+Vue.component('button-counter', {
+  data: () => ({
+    count: 0
+  }),
+  template: '<button @click="count++">Clicked {{ count }} times.</button>'
+})
+
+export default {
+  render(h) {
+    return h('button-counter')
+  }
+}
+```
+
+### 3.x Syntax
+
+In 3.x, with VNodes being context-free, we can no longer use a string ID to implicitly lookup registered components. Instead, we need to use an imported API:
+
+```js
+// 3.x
+import { h, resolveComponent } from 'vue'
+
+const ButtonCounter = resolveComponent('button-counter')
```

Comment:
I think you'd get a warning calling it like this:

> [Vue warn]: resolveComponent can only be used in render() or setup().

Reply:
Sorry, I will double-check the examples provided.